src/lib/evil/evil_private.h

```diff
@@ -21,6 +21,10 @@ extern "C" {
 #define EVIL_UNUSED
 #endif
 
+#ifdef HAVE_CONFIG_H
+# include "config.h"
+#endif /* HAVE_CONFIG_H */
+
```

Comment:
I think that it should be placed in `src/lib/evil/evil_sys_time.c`.

Reply:
Oh, you're right, when I recommended putting it there I could swear Eina was including `config.h` in `Eina.h` or `eina_private.h`, but turns out they include their "local" config (`eina_config.h`).

Reply:
Okay, I looked at it. As we need to see if meson set `HAVE_GETTIMEOFDAY` we need `config.h` at `evil_sys_time.h` too.

Given how `evil` deal with includes: includes should be done at `evil_private.h` and not at the specific header (such as `evil_sys_time.h`), that's why we have this right here, and as it is done at `evil_private.h` it doesn't need to be done at `.c` file, unless we want to explicitly it.